utility/BUILD.bazel

```diff
@@ -0,0 +1,53 @@
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
+load("@rules_jvm_external//:defs.bzl", "artifact")
+load("@dagger//:workspace_defs.bzl", "dagger_rules")
+
+package(default_visibility = ["//visibility:public"])
+
+kt_android_library(
+  name = "utility_lib",
+  custom_package = "org.oppia.util",
+  srcs = [
+      "src/main/java/org/oppia/util/accessibility/*.kt",
+      "src/main/java/org/oppia/util/caching/*.kt",
+      "src/main/java/org/oppia/util/data/*.kt",
+      "src/main/java/org/oppia/util/datetime/*.kt",
+      "src/main/java/org/oppia/util/gcsresource/*.kt",
+      "src/main/java/org/oppia/util/logging/*.kt",
+      "src/main/java/org/oppia/util/networking/*.kt",
+      "src/main/java/org/oppia/util/parser/*.kt",
+      "src/main/java/org/oppia/util/profile/*.kt",
+      "src/main/java/org/oppia/util/system/*.kt",
+      "src/main/java/org/oppia/util/threading/*.kt",
+      "src/main/java/org/oppia/util/extensions/*.kt",
+      "src/main/java/org/oppia/util/statusbar/*.kt",
+  ],
+  resource_files = glob(["src/main/res/**/*.xml", "src/main/res/**/*.png"]),
+  manifest = "src/main/AndroidManifest.xml",
```

Comment:
We should specify a separate manifest rather than reusing it, otherwise I think this might turn into unexpected issues.

Reply:
A separate manifest from what? Do you mean a Bazel specific manifest?

Reply:
No, I mean that only the app module should use the app module’s manifest. If we need a manifest for this module, we should create a new one.

Reply:
The manifest I am linking to here is the manifest under /utility/src/main/. Is that incorrect?

Reply:
Ah, you're right. I misread this. Please disregard. I think I also duplicated this same comment on several other PRs, so I suspect those will be quick to address. :) Thanks.